themes/index.js

```diff
@@ -268,6 +268,24 @@ const themes = {
     icon_color: "ff1aff",
     text_color: "8080ff",
     bg_color: "141439",
+  },
+  "rose-pine":{
```

Comment:
Please change the naming case to use underscores.

Reply:
Done